src/shared/PreviewNotAvailable.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const PreviewNotAvailable = () => (
+    <svg width="100" height="100" className="thumbnail">
+        <rect x="0" y="0" width="100" height="100" className="thumbnail" fill="#d2d2d2"/>
+        <text x="50%" y="40%" alignmentBaseline="middle" textAnchor="middle">Preview not</text>    
+        <text x="50%" y="60%" alignmentBaseline="middle" textAnchor="middle">available</text>    
+    </svg>
```

Comment:
@janakitti I was wondering whether you considered using `.svg` file instead?

It is indeed faster to load a _single_ svg using inline approach since it does not result in additional HTTP request being made. However, given that this svg could appear often in multiple places (rows), I believe it would be better to utilize `.svg` file so it gets cached. Indeed, with svg file, loading the image the first time will be slower due to the HTTP request to fetch the file; however, once cached, I believe it will be faster than loading inline svg multiple times. Lastly, by using a separate svg, styles are encapsulated, which could be useful to avoid CSS conflicts.

Let me know what are your thoughts about this. Open to discuss it further to find what would be the best approach for this.

Reply:
Just following up on the call we had to discuss this: I think we'll just stick with using inline SVGs for the time being. Although being able to cache SVG files is great for performance, we'll end up needing to embed fonts and minify the file, making it hard to edit in the future. In the average use case, we don't expect to see a large number of this graphic on the page at once, so I think it's a reasonable tradeoff to make.

Reply:
Sounds good